features/nanostack/sal-stack-nanostack/source/6LoWPAN/Thread/thread_network_data_storage.c

```diff
@@ -2343,7 +2343,8 @@ bool thread_nd_service_anycast_address_mapping_from_network_data(thread_network_
         }
         ns_list_foreach(thread_network_data_service_server_entry_t, curServiceServer, &curService->server_list) {
             *rlocAddress = curServiceServer->router_id;
-            return true;
```

Comment:
We would like to fix this by removing the inner loop and using ns_list_get_first to read the RLOC. Can this wait till Mbed OS 5.11?